src/styles/blocks/card.scss

```
@@ -0,0 +1,77 @@
+.card{
+  box-sizing: border-box;
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid #F3F3F3;
+  background: $bg-color;
+  padding: 32px 16px 16px;
+  transition:  0.3s transform;
+  margin: 0 auto;
+
+
+  &:hover{
+    transform: scale(1.2);
+    transition:  0.3s transform;
+  }
+  &__link{
+    margin-bottom: 40px;
+  }
+
+  &__title{
+    display:block;
+    color: $header-link-color;
+    font-weight: 500;
+    line-height: 18px;
+    margin-bottom: 4px;
+    transition:  0.3s color;
+
+    &:hover{
+      color:$color-card-title-hover;
+      transition:  0.3s color;
+    }
+  }
+
+  &__product-code{
+    color:$text-color;
+    font-size: 10px;
+    line-height: 14px;
+    align-self: flex-start;
+    margin-bottom: 16px;
+    font-weight: 400;
+  }
+
+  &__raiting{
+    margin-bottom: 24px;
+  }
+
+  &__price{
+    margin-bottom: 16px;
+  }
+
+  &__button{
+    box-sizing: border-box;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 100%;
+    height: 40px;
+    border-radius: 5px;
+    background-color: $color-blue;
+    color: $bg-color;
+    font-size: 14px;
+    font-weight: 700;
+    text-align: center;
+    line-height: 16px;
+    text-transform: uppercase;
+    outline: none;
+    border: none;
+    text-decoration: none;
+
```

put it here
don't use it like that, because it's the same as transition: all 0.3s
It's too hard for the browser
Check this
https://github.com/mate-academy/layout_catalog_hovers/blob/master/checklist.md

Create a variable for transition duration
For example
```suggestion
    transition: color $transitionDuration, transform $transitionDuration;
```